spyder/utils/tests/test_programs.py

```diff
@@ -6,10 +6,17 @@
 """Tests for programs.py"""
 
 import os
-
 import pytest
 
-from spyder.utils.programs import run_python_script_in_terminal
+from spyder.utils.programs import (run_python_script_in_terminal,
+                                   is_python_interpreter)
+
+if os.name == 'nt':
+    VALID_INTERPRETER = r'%PYTHON%\bin\python.exe'
```

Comment:
Sorry, my code was a simple draft of what you should do, not the right solution (that's why tests are still failing :-). So this should be

```python
python_dir = os.environ['PYTHON']
VALID_INTERPRETER = os.path.join(python_dir, 'bin', 'python.exe')
```